Description: Remove makefile targets
  Remove all targets from the build. This is in preparation for patch development.
  Modify statements for install target to work with Debian kbuild.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: not-needed
Last-Update: <2018-09-18>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -7,8 +7,8 @@
 ifneq ($(KERNELRELEASE),)
 #kbuild part of makefile
 
-CFLAGS_MODULE += -DUSE_PI_LED_ENABLE=1 -DIFS_RH76
-obj-y := rdmavt/ hfi1/ ib_qib/ ib_ipoib/
+CFLAGS_MODULE += -DUSE_PI_LED_ENABLE=1 -DIFS_DEB8
+obj-y :=
 
 else
 #normal makefile
@@ -21,6 +21,6 @@
 	$(MAKE) -C $(KDIR) M=$$PWD clean
 
 install:
-	$(MAKE) INSTALL_MOD_DIR=updates -C $(KDIR) M=$$PWD modules_install
+	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=updates/ifs-kernel-updates -C $(KDIR) M=$$PWD modules_install
 
 endif
